instance/tests/models/test_openedx_monitoring_mixins.py

```diff
@@ -126,8 +125,8 @@ def test_enable_monitoring_for_pre_new_relic_alerts_instances(self, mock_newreli
         """
         instance = OpenEdXInstanceFactory()
         monitor_ids = [str(uuid4()) for i in range(len(instance._urls_to_monitor))]
-        for index, monitored_url in enumerate(instance._urls_to_monitor.keys()):
-            NewRelicAvailabilityMonitor.objects.create(instance=instance, pk=monitor_ids[index])
+        for monitor_id in monitor_ids:
```

Comment:
Does this have the same effect as the previous code?
It seems like I let this pass on the review.

Reply:
I think yes (also, the test passes): `monitored_url` wasn't used, only the index is used, and `index` is just 0 1 2 3 4… to get the IDs. Better to directly iterate over the IDs.